examples/README.md

```diff
@@ -999,12 +999,14 @@ See [README](../community/examples/flux-framework/README.md)
 
 [flux-cluster.yaml]: ../community/examples/flux-framework/flux-cluster.yaml
 
-### [hpc-slurm-legacy-sharedvpc.yaml] ![community-badge] ![deprecated-badge]
+### [hpc-slurm-legacy-sharedvpc.yaml] ![community-badge]
 
 This blueprint demonstrates the use of the Slurm and Filestore modules in
 the service project of an existing Shared VPC.  Before attempting to deploy the
 blueprint, one must first complete [initial setup for provisioning Filestore in
-a Shared VPC service project][fs-shared-vpc].
+a Shared VPC service project][fs-shared-vpc]. Also since this blueprint doesn't
+use external ip for compute nodes, one must need to [set up cloud nat][cloudnat]
```

Comment:
Grammer:
s/use external ip for compute nodes/use external IPs for the compute nodes/

Reply:
Done.